### PR TITLE
sync prettier and ESLint settings

### LINKED ISSUE #1230: make prettier and ESLing settings 1-1 compatible

### PR DESCRIPTION
**Pull request by LetsPoke**
closes #1230

### CODE REVIEW

File: hms-app/.eslintrc.json [hunk 1]
```diff
@@ -24,11 +24,17 @@
         "prettier"
     ],
     "rules": {
+        "indent": ["error", 2],
```
Comment (verzano) [inline hms-app/.eslintrc.json:27]: no `comma-dangle` in this one

Comment (LetsPoke) [reply hms-app/.eslintrc.json:27]: thats correct

Comment (LetsPoke) [reply hms-app/.eslintrc.json:27]: thats also correct

Comment (LetsPoke) [reply hms-app/.eslintrc.json:27]: Jokes aside 
in the frontend setting of prettier its set to ""trailingComma": "es5","
but could not find a fitting setting für comma-dangle so I leave it out as I assume everyone in frontend will use prettier

**Review by verzano — APPROVED** [review]

**Pull request merged**